pkg/utils/watchJob.go

```diff
@@ -137,8 +136,8 @@ func (engineDetails EngineDetails) DeleteJobAccordingToJobCleanUpPolicy(experime
 			PropagationPolicy: &deletePolicy,
 		})
 		if deleteJob != nil {
-			return errors.Wrapf(err, "Unable to delete ChaosExperiment Job Name: %v, in namespace: %v, due to error: %v", experiment.JobName, experiment.Namespace, err)
+			return "", errors.Wrapf(err, "Unable to delete ChaosExperiment Job Name: %v, in namespace: %v, due to error: %v", experiment.JobName, experiment.Namespace, err)
 		}
 	}
-	return nil
+	return expEngine.Spec.JobCleanUpPolicy, nil
```

Comment:
Any particular benefit of these changes in this function? We return `"" ` in case of errors and a known spec value in case of success ..

Reply:
Got it - in the context of event usage.